src/mac/mac_messages.h

```diff
@@ -40,18 +42,26 @@ typedef enum {
   ul_mcs_info,
   timing_advance,
   session_end,
+  ul_data_ack,
   dl_data = 7,
   ul_req = 9,
   channel_quality,
   keepalive,
   control_ack,
   mcs_chance_req,
+  dl_data_ack,
   ul_data = 15
 } CtrlID_e;
```

Comment:
Please assign enum values explicitly to avoid potential confusion due to different enummeration when using different compilers.

Reply:
Fixed in https://github.com/HAMNET-Access-Protocol/HNAP4PlutoSDR/pull/37/commits/5ffd0417ecce7a9df6af43c2844005a180fcd570